crates/shared/src/price_estimation/native_price_cache.rs

```diff
@@ -227,6 +227,22 @@ impl UpdateTask {
 }
 
 impl CachingNativePriceEstimator {
+    /// Convert from normalized price to floating point price
+    fn from_normalized_price(price: U256) -> Option<f64> {
+        // Convert U256 to f64
+        let price_in_eth = price.to_f64_lossy();
+
+        // Divide by 1e18 to reverse the multiplication by 1e18
+        let normalized_price = price_in_eth / 1e18;
```

Comment:
This normalization assumes that the reference token always has 18 decimals. What would be more correct is to pass the `native_token` into the function and use its reference price to normalize the other prices with.

Reply:
Bummer that it's also harcoded there. :/
To not open another can of worms could you please define this as a constant in `shared/price_estimation/native.rs` so that the `autopilot` can reference that there as well. That way we at least can't forget to keep these values in sync.

Reply:
Sounds good